### DISCUSSION TITLE
Development Environment

### DISCUSSION THREAD

**Discussion by jaskier2** [microsoft/semantic-link-labs#251] [Q&A]
Hi,
 
I am wondering what development setup is used to test/develop this library?

Is there a recommended way to run code against a fabric environment from a local IDE? 

Is there a dev container we could use or I have read about livy endpoints but there doesn't seem to be any officicial documentation on them yet.

Thank you.

**Comment by m-kovalsky** [comment]
We do not support running the library from a localbox. Generally testing is done by building the .whl file and adding it to a Fabric notebook.
